app/assets/stylesheets/shared/form.scss

```diff
@@ -39,6 +39,25 @@
   }
 }
 
+// bootstrap5-tags need help
```

Comment:
Found/added this [bootstrap5-tags](https://github.com/lekoala/bootstrap5-tags) library.  Mostly works and we can add autocomplete ... but needed (needs?) some styling help.